hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/TestMiniChaosOzoneCluster.java

```diff
@@ -90,6 +99,7 @@ public static void init() throws Exception {
         .setNumOzoneManagers(numOzoneManagers)
         .setFailureService(failureService)
         .setOMServiceID(omServiceID)
+        .setNumDatanodesDirs(numDataVolumes)
```

Comment:
Can we make the naming consistent?

 * `numOfDatanodesDirs`
 * `setNumDatanodesDirs` (implies `numDatanodesDirs` is being set, note no `Of`)
 * `numDataVolumes`